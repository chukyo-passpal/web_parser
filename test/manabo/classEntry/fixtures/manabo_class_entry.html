<div class="modal-dialog">
    <div class="modal-content">
        <div class="modal-header">
            <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true"
                    class="glyphicon glyphicon-remove"></span><span class="sr-only">Close</span></button>
            <i class="mark"></i>出席表
        </div>
        <div class="modal-body">
            <div class="table-responsive">
                <table class="table-default">
                    <thead>
                        <tr>
                            <th>ディレクトリ</th>
                            <th>講義日</th>
                            <th>出欠</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>第3回 Javaプログラムを動かす</td>
                            <td>
                            </td>
                            <td>
                                欠席
                            </td>
                        </tr>
                        <tr>
                            <td>第4回 staticキーワード、基本型</td>
                            <td>
                            </td>
                            <td>
                                欠席
                            </td>
                        </tr>
                        <tr>
                            <td>第6回 制御構造</td>
                            <td>
                            </td>
                            <td>
                                欠席
                            </td>
                        </tr>
                        <tr>
                            <td>第8回 値による呼び出し、可変引数</td>
                            <td>
                            </td>
                            <td>
                                欠席
                            </td>
                        </tr>
                        <tr>
                            <td>第9回 コマンドライン引数、Mathクラス、オーバーロード</td>
                            <td>
                            </td>
                            <td>
                                出席
                            </td>
                        </tr>
                        <tr>
                            <td>第10回 コンストラクタ</td>
                            <td>
                            </td>
                            <td>
                                出席
                            </td>
                        </tr>
                    </tbody>
                </table>
            </div>
        </div>
    </div>
</div>
