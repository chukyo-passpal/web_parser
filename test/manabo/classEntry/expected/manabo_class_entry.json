{
  "rows": [
    {
      "directory": "第3回 Javaプログラムを動かす",
      "lectureDate": null,
      "status": "欠席"
    },
    {
      "directory": "第4回 staticキーワード、基本型",
      "lectureDate": null,
      "status": "欠席"
    },
    {
      "directory": "第6回 制御構造",
      "lectureDate": null,
      "status": "欠席"
    },
    {
      "directory": "第8回 値による呼び出し、可変引数",
      "lectureDate": null,
      "status": "欠席"
    },
    {
      "directory": "第9回 コマンドライン引数、Mathクラス、オーバーロード",
      "lectureDate": null,
      "status": "出席"
    },
    {
      "directory": "第10回 コンストラクタ",
      "lectureDate": null,
      "status": "出席"
    }
  ]
}